v1/messages/src/requestHandlers.ts

```diff
@@ -107,25 +107,20 @@ function parseSlugFromURI(uri: string, resource: string): string {
   return "";
 }
 
-function parseAuthorizationHeader(auth: string): string | null {
-  if (auth === undefined) {
-    return null;
-  }
+function parseBasicAuth(auth: string): string | null {
```

Comment:
Your signature is wrong, auth is `string | null` according to the value you pass in from index.ts. So you are missing a null check here